xml_converter/src/state_structs/xml_reader_state.hpp

```diff
@@ -1,11 +1,9 @@
 #pragma once
 
-#include <map>
 #include <string>
 
 class Category;
 
 struct XMLReaderState {
-    std::string xml_filedir;
-    std::map<std::string, Category>* marker_categories;
```

Comment:
I think we need this data to do some validations against some fields that reference categories, even if we are not using it right now.